src/Generator/TraitUsageGenerator.php

```diff
@@ -65,6 +67,70 @@ public function getUses()
     }
 
     /**
+     * @param $use
+     * @return bool
+     */
+    public function hasUse($use)
+    {
+        foreach ($this->uses as $key => $value) {
+            $parts = explode(' ', $value);
```

Comment:
I wish `$this->uses` wasn't `protected`: makes it so hard to migrate away from it :-(